lib/PrintAsClang/ModuleContentsWriter.cpp

```diff
@@ -240,6 +240,11 @@ class ModuleWriter {
   }
   
   void forwardDeclare(const EnumDecl *ED) {
+    // FIXME: (tongjie) do we need this?
+    if (outputLangMode == swift::OutputLanguageMode::Cxx) {
+      return;
+    }
+
```

Comment:
I simply skip this `forwardDeclare` for now because there's no forward declaration function for Swift struct in this file.

Reply:
For now yes, I want to overhaul the forward decals for c++ in the future so we will fix it up later